doc/source/conf.py

```diff
@@ -157,6 +160,98 @@
     }
 
 # -- Jinja context configuration ---------------------------------------------
+
+def zip_directory(directory_path: pathlib.Path, zip_filename: pathlib.Path, ignore_patterns=None):
+    """Compress a directory using ZIP.
+
+    Parameters
+    ----------
+    directory_path : ~pathlib.Path
+        Directory to compress.
+    zip_filename : ~pathlib.Path
+        Output file path.
+    ignore_patterns : list
+        List of Unix-like pattern to ignore.
+
+    """
+    if ignore_patterns is None:
+        ignore_patterns = []
+
+    if not zip_filename.suffix == ".zip":
+        zip_filename = zip_filename.with_suffix(".zip")
+
+    with zipfile.ZipFile(zip_filename, "w", zipfile.ZIP_DEFLATED) as zipf:
+        for file_path in directory_path.rglob("*"):
+            if file_path.is_file():
+                if any(fnmatch.fnmatch(file_path.relative_to(directory_path), pattern) for pattern in ignore_patterns):
+                    continue
+
+                relative_path = file_path.relative_to(directory_path)
+                zipf.write(file_path, relative_path)
+
+
+def get_sha256_from_file(filepath: pathlib.Path):
+    """Compute the SHA-256 for a file.
+
+    Parameters
+    ----------
+    filepath : ~pathlib.Path
+        Desired file.
+
+    Returns
+    -------
+    str
+        String representing the SHA-256 hash.
+
+    """
+    sha256_hash = hashlib.sha256()
+    with open(filepath, "rb") as file:
+        while chunk := file.read(8192):
+            sha256_hash.update(chunk)
+    return sha256_hash.hexdigest()
+
+
+def get_file_size_in_mb(file_path):
+    """
+    Compute the size of a file in megabytes.
+
+    Parameters
+    ----------
+    file_path : str or Path
+        The path to the file whose size is to be computed.
+
+    Returns
+    -------
+    float
+        The size of the file in megabytes.
```

Comment:
```suggestion
        Size of the file in megabytes.
```
Don't include a leading article in brief descriptions.